stable/kommander-thanos/templates/_helpers.tpl

```diff
@@ -31,6 +31,13 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Create a truncated fullname suitable for addon names.
+*/}}
+{{- define "kommander-thanos.addon-fullname" -}}
```

Comment:
`fullname` is a bit misleading for a truncated name....  how about `short-name-prefix`?